diff --git a/node_modules/@web/dev-server/dist/startDevServer.js b/node_modules/@web/dev-server/dist/startDevServer.js
index 402a461..fda0ef3 100644
--- a/node_modules/@web/dev-server/dist/startDevServer.js
+++ b/node_modules/@web/dev-server/dist/startDevServer.js
@@ -14,7 +14,8 @@ const openBrowser_1 = require("./openBrowser");
  */
 async function startDevServer(options = {}) {
     var _a;
-    const { config: extraConfig, readCliArgs: readCliArgsFlag = true, readFileConfig: readFileConfigFlag = true, configName, autoExitProcess = true, logStartMessage = true, argv = process.argv, } = options;
+    const { config: extraConfig, readCliArgs: readCliArgsFlag = true, readFileConfig: readFileConfigFlag = true, configName, autoExitProcess = true, logStartMessage = true, argv = process.argv, 
+        externalServer, } = options;
     try {
         const cliArgs = readCliArgsFlag ? (0, readCliArgs_1.readCliArgs)({ argv }) : {};
         const rawConfig = readFileConfigFlag
@@ -29,7 +30,7 @@ async function startDevServer(options = {}) {
         });
         config.plugins = (_a = config.plugins) !== null && _a !== void 0 ? _a : [];
         config.plugins.unshift(loggerPlugin);
-        const server = new dev_server_core_1.DevServer(config, logger);
+        const server = new dev_server_core_1.DevServer(config, logger, undefined, externalServer);
         if (autoExitProcess) {
             process.on('uncaughtException', error => {
                 /* eslint-disable-next-line no-console */
